cfme/tests/configure/test_rest_access_control.py

```diff
@@ -66,7 +66,7 @@ def test_edit_tenants(self, appliance, tenants, multiple):
         assert tenants_len == len(edited)
         for index in range(tenants_len):
             record, _ = wait_for(
-                lambda: collection.find_by(name=new[index]['name']),
+                lambda: collection.find_by(name=new[index]['name']) or False,
```

Comment:
Thanks for comment, I was not able to come up with nice `fail_condition` in this situation. When nothing is found the `collection.find_by` returns empty `SearchResult` which is false. So I would need to use something like `fail_condition=lambda result: not result` if I'm not mistaken.